op-e2e/actions/proofs/bad_tx_in_batch_test.go

```diff
@@ -5,12 +5,16 @@ import (
 
 	actionsHelpers "github.com/ethereum-optimism/optimism/op-e2e/actions/helpers"
 	"github.com/ethereum-optimism/optimism/op-e2e/actions/proofs/helpers"
-	"github.com/ethereum-optimism/optimism/op-program/client/claim"
-	"github.com/ethereum/go-ethereum/common"
 	"github.com/ethereum/go-ethereum/core/types"
+	"github.com/ethereum/go-ethereum/crypto"
 	"github.com/stretchr/testify/require"
 )
 
+const (
+	BadSignature int64 = iota
+	NotEnoughBalance
+)
+
 func runBadTxInBatchTest(gt *testing.T, testCfg *helpers.TestCfg[any]) {
```

Comment:
```suggestion
func runBadTxInBatchTest(gt *testing.T, testCfg *helpers.TestCfg[int64]) {
```
I'm surprised this works without this because I would have assumed that the `testCfg.Custom == NotEnoughBalance` checks don't work because the left side is of type `any`.